package.json

```diff
@@ -33,7 +33,7 @@
     "@nestjs/platform-express": "^10.3.7",
     "@nestjs/platform-socket.io": "^10.3.7",
     "@nestjs/schedule": "^3.0.0",
-    "@nestjs/swagger": "^7.3.1",
+    "@nestjs/swagger": "^7.4.0",
```

Comment:
any reason for updating the version here?

Reply:
I guess this happened when I tried to re-install Swagger without knowing it was here already